tensorflow-notebook/Dockerfile

```diff
@@ -10,9 +10,10 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 # Fix: https://github.com/koalaman/shellcheck/wiki/SC3014
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Install Tensorflow
+# Install Tensorflow with pip
 RUN mamba install --yes \
-    'tensorflow' && \
+    'pip' && \
     mamba clean --all -f -y && \
```

Comment:
I've just checked `mamba list` and `pip` is already there, so we don't need to install it here, I suppose.